tools/code-owners-parser/custom-tests.yml

```diff
@@ -0,0 +1,6 @@
+steps:
+  - pwsh: |
+      $output = dotnet run --project "tools\code-owners-parser\Azure.Sdk.Tools.RetrieveCodeOwners\Azure.Sdk.Tools.RetrieveCodeOwners.csproj" `
+        --code-owner-file-path "$(Build.SourcesDirectory)/tools/code-owners-parser/Azure.Sdk.Tools.RetrieveCodeOwners.Tests/CODEOWNERS" --target-directory "sdk"
+      $output | ConvertFrom-Json
```

Comment:
I think it's still worth double checking a zero exit code here just to be thorough. I know the JSON conversion check will handle all cases like this right now, but I could see some case in the future where, based on some changes, we console write the JSON and then fail for some subsequent operation.

Reply:
I double check about the nonzero exit scenario. It did fail the pipeline.
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1203323&view=results